projects/storefrontapp/src/app/app.module.ts

```diff
@@ -49,8 +80,8 @@ if (!environment.production) {
       featureModules: {
         productConfiguratorRulebased: {
           module: () =>
-            import('@spartacus/product-configurator/rulebased').then(
```

Comment:
There are many feature modules inside `features` folder, it is better to move CPQ stuff into there.

Reply:
picked

Reply:
That's a known issue, for some reasons the SPA CI was not able to handle the previously existing feature toggle, so @Platonn created #10883